classes/ObjectModel.php

```diff
@@ -1727,7 +1727,7 @@ public function isMultishop()
      */
     public function isMultiShopField($field)
     {
-        return isset($this->def['fields'][$field]) && isset($this->def['fields'][$field]['shop']) && $this->def['fields'][$field]['shop'];
+        return isset($this->def['fields'][$field], $this->def['fields'][$field]['shop']) && $this->def['fields'][$field]['shop'];
```

Comment:
you can simplify this one with `isset($this->def['fields'][$field]['shop'])`

Reply:
And why not directly
```suggestion
        return !empty($this->def['fields'][$field]['shop']);
```

Reply:
even better!